kernel/src/lib/nuumshell/shell.cr

```diff
@@ -0,0 +1,66 @@
+require "./apps/*"
+
+class NuumShell
+  def initialize
+    @user = "root"
+    @pass = "root"
```

Comment:
Is a password needed? At the moment we don't have a filesystem, and a password is essentially useless. :P